openreview/arr/process/emergency_load_process.py

```diff
@@ -113,7 +113,7 @@ def process(client, edit, invitation):
         original_load = 0
     else:
         original_load = cmp_edges[0].weight
-    emergency_load = edit.note.content['emergency_load']['value']
+    emergency_load = edit.note.content.get('emergency_load', {}).get('value', 0)
```

Comment:
is there a way to reproduce this error in the tests?

Reply:
There are already cases where the emergency load is not present but they're being caught by the pre-process